### DISCUSSION TITLE
Sequential vs. non-sequential optimization (PEST++-IES)

### DISCUSSION THREAD

**Discussion by Eakara21** [usgs/pestpp#314] [Q&A] [answered]
Hello All,
I am using PEST++-IES to optimize a Modflow6 hydraulic conductivity field.
My optimization targets include measured "steady state" heads and pumping test data.
The optimization targets were subdivided into 2 groups. Group1 include steady state heads and half of the pumping test data.
Group2 consists of the remaining half of the pumping test data.
The first optimization uses the prior hydraulic conductivity ensemble and Group1 data. 
The posterior ensemble from the first optimization is then optimized against Group2 data.

I want to know if the posterior ensemble from this sequential approach will be similar to an optimization where all the optimization targets (Group1+Group2) are used at once.

Thanks,
-Romeo

**Comment by jtwhite79** [comment]
Really good question.  In theory, the posterior ensemble from the first assimilation need not be similar to the posterior ensemble from the second assimilation, meaning that the posterior ensemble that best fits the group2 data might not fit the steady state and group1 data if you go back and check it.  If you want to (try to) fit all the data at once but still retain the 2 steps, you could keep the group1 and steady state obs active (non-zero weighted) during the second step, but reset their observed values to the final simulated values from the first step - essentially trying to keep them from changing during the 2nd step.

**Reply by Eakara21** [reply]
Thanks for the answer. Can I take from you explanation that, in general, sequential and non-sequential optimization will yield similar results.

**Reply by jtwhite79** [reply, marked as answer]
In general sequential and batch formulations will not yield similar results - sequential is heavily weighted toward the last set of data that where assimilated whereas batch tries to fit all available data simultaneously.

**Reply by Eakara21** [reply]
Thanks very much.
